### PR TITLE
use new get_node ($) in intro

### PR DESCRIPTION
**Pull request by toger5**

### CODE REVIEW

File: about/introduction.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Introduction
 ::
 
     func _ready():
-        get_node("Label").set_text("Hello world!")
+        $Label.set_text("Hello world!")
```
Comment (vnen) [inline about/introduction.rst]: I think it would be better to also use members, so `$Label.text = "Hello World!"`.

Comment (toger5) [reply about/introduction.rst]: lol obviously...

**Comment by toger5** [comment]
updated...

**Comment by cbscribe** [comment]
I like it!

**Pull request merged**